clippy_lints/src/implicit_return.rs

```diff
@@ -90,6 +90,12 @@ impl Pass {
                 if let Some(expr) = &block.expr {
                     Self::expr_match(cx, expr);
                 }
+                // only needed in the case of `break` with `;` at the end
+                else if let Some(stmt) = block.stmts.last() {
```

Comment:
Clippy suggests to refactor this, as it's now exactly the same match body as `ExprKind::Block(block, ..)`

Reply:
Yeah I'm looking at it right now, I'm still too dumb to run the same tests as the CI locally it seems. Working on it.